docs/cloudfoundry/cfctlr-app-install.rst

```diff
@@ -65,24 +65,77 @@ Include the following in the :code:`env.BIGIP_CTLR_CFG.bigip` section of the man
 Add BIG-IP Health Monitors
 ``````````````````````````
 
-You can attach any existing BIG-IP health monitors to the virtual server(s) created by the |cfctlr|.
-For example: To attach an existing BIG-IP HTTP monitor called "http.get" that resides in the :code:`/Common` partition to the virtual server, add the following to your Application Manifest:
-
-.. code-block:: yaml
-   :caption: Excerpt from example manifest with health monitor defined
-   :emphasize-lines: 11
-
-   BIGIP_CTLR_CFG: |
-             bigip:
-               url: https://bigip.example.com
-               user: myBigipUsername
-               pass: myBigipPassword
-               partition:
-                 - cf
-               balance: round-robin
-               verify_interval: 30
-               external_addr: 192.168.1.1
-               health-monitors: /Common/http.get
+Remember, settings provided in the :code:`bigip` section are global. If you want to create health monitors on a per-Route basis, see
+
+-  Attach an existing BIG-IP health monitors to the virtual server:
+
+   .. code-block:: yaml
+      :caption: Excerpt from example manifest with health monitor defined
+      :emphasize-lines: 11
+
+      BIGIP_CTLR_CFG: |
+                bigip:
+                  url: https://bigip.example.com
+                  user: myBigipUsername
+                  pass: myBigipPassword
+                  partition:
+                    - cf
+                  balance: round-robin
+                  verify_interval: 30
+                  external_addr: 192.168.1.1
+                  health-monitors:
+                  - /Common/http.get
+
+   This attaches an existing BIG-IP health monitor called "http.get", which resides in the :code:`/Common` partition.
+
+-  Create and attach a new health monitor to the virtual server:
+
+   .. code-block:: yaml
+      :caption: Excerpt from example manifest with health monitor defined
+      :emphasize-lines: 11-16
+
+      BIGIP_CTLR_CFG: |
+                bigip:
+                  url: https://bigip.example.com
+                  user: myBigipUsername
+                  pass: myBigipPassword
+                  partition:
+                    - cf
+                  balance: round-robin
+                  verify_interval: 30
+                  external_addr: 192.168.1.1
+                  health-monitors:
+                  - name: hm-1
+                    type: http
+                    interval: 12
+                    timeout: 5
+                    send: hello
+
+- You can also use an existing BIG-IP health monitor and create a new one:
```

Comment:
This won't work in the main config, this must be a pre existing health monitor, creating a health monitor is only supported through the plans